modules/nf-core/trimgalore/main.nf

```diff
@@ -1,6 +1,8 @@
 process TRIMGALORE {
     tag "${meta.id}"
     label 'process_high'
+    // ~3GB per actual core used (15 cores * 3GB + 3GB buffer)
+    resourceLimits [memory: 48.GB, cpus: 15]
```

Comment:
I think this syntax will not work with Nextflow, though it does work with the language server.

In any case the "idiomatic" syntax would be:
```groovy
resourceLimits memory: 48.GB, cpus: 15
```